package.json

```diff
@@ -21,10 +21,10 @@
   },
   "license": "MIT",
   "scripts": {
-	"build": "yarn build-main && yarn build-renderer",
-	"build-main": "tsc",
-	"build-renderer": "yarn webpack --config webpack.config.js",
-	"watch-renderer": "yarn build-renderer --watch",
+    "build": "yarn build-main && yarn build-renderer",
+    "build-main": "tsc",
+    "build-renderer": "yarn webpack --config webpack.config.js",
+    "watch-renderer": "yarn build-renderer --watch",
```

Comment:
It works!

Reply:
![giphy](https://user-images.githubusercontent.com/21110659/99091768-efe43f80-2595-11eb-8c21-c0f3ed8712fb.gif)